lib/interface/cli/defaults.js

```diff
@@ -13,6 +13,8 @@ const DEFAULTS = {
     MAX_CONSECUTIVE_ERRORS_LIMIT: 10,
     CODEFRESH_PATH: path.resolve(homedir(), '.Codefresh'),
     ENGINE_IMAGE: process.env.ENGINE_IMAGE || 'codefresh/engine:master',
+    CLI_INTERNAL_CONFIG: 'CLI_INTERNAL_CONFIG',
+    CF_API_ENCRYPTED_SUPPORTED_VERSION: '21.221.14',
```

Comment:
the name of this var is a bit unclear - I can suggest the following:
`MIN_API_VERSION_FOR_ENCRYPTED_VARS_SUPPORT_IN_RUN_CMD`